### PR TITLE
Fixes some Kapulimbs bugs

### LINKED ISSUE #65920: Husked bodies do not appear husked
Reporting client version: 514.1582

## Round ID:
[181044](https://scrubby.melonmesa.com/round/181044)

## Testmerges:
- [Refactor and improve antimagic to be more robust](https://www.github.com/tgstation/tgstation/pull/64124)

## Reproduction:
image
image

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
closes #65920

## Why It's Good For The Game
Take a guess dumbass

## Changelog

:cl:
fix: Husks render again
fix: Lizards, Moths, and Podpeople character creation icons fixed
/:cl:

### CODE REVIEW

File: code/modules/surgery/bodyparts/_bodyparts.dm [hunk 1]
```diff
@@ -717,6 +717,17 @@
 			limb.overlays += limb_em_block
 		return
 
+	//HUSK SHIIIIT
```
Comment (Mothblocks) [inline code/modules/surgery/bodyparts/_bodyparts.dm:720]: Useless comments, remove it

File: code/modules/surgery/bodyparts/_bodyparts.dm [hunk 2]
```diff
@@ -717,6 +717,17 @@
...
+	if(is_husked)
+		limb.icon = icon_husk
+		limb.icon_state = "[husk_type]_husk_[body_zone]"
+		icon_exists(limb.icon, limb.icon_state, TRUE) //Prints a stack trace on the first failure of a given iconstate.
+		if(aux_zone) //Hand shit
+			aux = image(limb.icon, "[husk_type]_husk_[aux_zone]", -aux_layer, image_dir)
+			. += aux
+		return
+	//END HUSK SHIIIIT
```
Comment (Mothblocks) [inline code/modules/surgery/bodyparts/_bodyparts.dm:729]: Useless comment

File: code/modules/surgery/bodyparts/_bodyparts.dm [hunk 3]
```diff
@@ -717,6 +717,17 @@
 			limb.overlays += limb_em_block
 		return
 
+	//HUSK SHIIIIT
+	if(is_husked)
+		limb.icon = icon_husk
+		limb.icon_state = "[husk_type]_husk_[body_zone]"
+		icon_exists(limb.icon, limb.icon_state, TRUE) //Prints a stack trace on the first failure of a given iconstate.
```
Comment (Mothblocks) [inline code/modules/surgery/bodyparts/_bodyparts.dm:724]: Named parameter for TRUE

File: code/modules/surgery/bodyparts/_bodyparts.dm [hunk 4]
```diff
@@ -717,6 +717,17 @@
...
+		if(aux_zone) //Hand shit
+			aux = image(limb.icon, "[husk_type]_husk_[aux_zone]", -aux_layer, image_dir)
+			. += aux
+		return
```
Comment (Mothblocks) [inline code/modules/surgery/bodyparts/_bodyparts.dm:728]:
```suggestion
		return .
```

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/surgery/bodyparts/_bodyparts.dm (hunk 1 above)
Comment (Kapu1178) [reply code/modules/surgery/bodyparts/_bodyparts.dm:720]: I very much like having comments breaking apart chunks of bodypart render code, it makes it significantly more readable.

Comment (Mothblocks) [reply code/modules/surgery/bodyparts/_bodyparts.dm:720]: I disagree but don't care enough to block the PR on it

**Pull request merged**